altinn/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/altinn/TilgangLoeser.kt

```diff
@@ -58,7 +58,7 @@ class TilgangLoeser(
         val requestTimer = requestLatency.startTimer()
         runCatching {
             runBlocking {
-                altinnClient.harRettighetForOrganisasjon(behov[Key.FNR].asText(), behov[Key.ORGNRUNDERENHET].asText())
+                altinnClient.harRettighetForOrganisasjon(behov.jsonMessage[Key.FNR.str].asText(), behov.jsonMessage[Key.ORGNRUNDERENHET.str].asText())
```

Comment:
Det funker vel fortsatt med behov[Key.FNR] - så hvorfor introdusere denne formen som er lengre å skrive? 
Det er i tillegg enda en metode som gjør det samme: behov.get(Key.FNR)

Reply:
Grunnen til endringen er at jeg på sikt mener vi ikke bør ha `behov[Key.FNR]`. Det skjuler bruker av `jsonMessage`-feltet, som er en av de mest lunefulle typene vi bruker i dag. Det er også i overkant "magisk" for min smak at en klasse med diverse felter bruker en Map-lignende måte å lese verdier fra en av feltene sine (som er et slags Map). 

`behov.get(Key.FNR)` bruker vel samme funksjon som `behov[Key.FNR]`? Tror IntelliJ vil foreslå å bytte til sistnevnte i de tilfellene.